packages/constants/src/constants.ts

```diff
@@ -68,59 +103,62 @@ export type TAppNotification<
     ? TAppNotificationValuesUnexpectedError
     : never;
 };
-export const NOTIFICATION_KINDS_SIDE: {
-  [key: string]: TAppNotificationKindSide;
-} = {
-  error: 'error',
-  warning: 'warning',
-  info: 'info',
-  success: 'success',
-};
-export const NOTIFICATION_KINDS_GLOBAL: {
-  [key: string]: TAppNotificationKindGlobal;
-} = {
-  ...NOTIFICATION_KINDS_SIDE,
-  'unexpected-error': 'unexpected-error',
-};
-export const NOTIFICATION_KINDS_PAGE: {
-  [key: string]: TAppNotificationKindPage;
-} = {
-  ...NOTIFICATION_KINDS_GLOBAL,
-  'api-error': 'api-error',
-};
+export type TAppNotificationGlobal = TAppNotification<{
```

Comment:
We can also construct specific types for each notification domain, which is necessary to be able to differentiate between the notifications when we handle them as a list.